locales/en.json

```
@@ -261,6 +261,7 @@
   "NFTPortal.button.create": "Create",
   "NFTPortal.button.collect": "Collect Now",
   "NFTPortal.button.convertToIscn": "Convert Content to Writing NFT",
+  "NFTPortal.button.download.iscn": "Download iscn.json",
```

I think the icon is enough
```suggestion
  "NFTPortal.button.download.iscn": "iscn.json",
```